standard/expressions.md

```diff
@@ -4443,6 +4443,16 @@ When recognising an *anonymous_function_body* if both the *null_conditional_invo
 
 > *Note*: The overlapping of, and priority between, alternatives here is solely for descriptive convenience; the grammar rules could be elaborated to remove the overlap. ANTLR, and other grammar systems, adopt the same convenience and so *anonymous_function_body* has the specified semantics automatically.
 
+> *Note*: When treated as an *expression*, a syntaxtic form such as `x?.M()` would be an error if the result type of `M` is `void` [§11.7.11](expressions.md#11711-null-conditional-element-access). But when treated as a *null_conditional_invocation_expression*, the result type is permitted to be `void`.
```

Comment:
Typo, I think the Standard normally would put the x-ref in parentheses, e.g. ([§11.7.11](expressions.md#11711-null-conditional-element-access))

Reply:
And I assume that syntaxtic should be syntactic?